src/PayloadParser.h

```diff
@@ -35,7 +35,7 @@ namespace snowcrash {
     };
 
     /** Request matching regex */
-    const char* const RequestRegex = "^[[:blank:]]*[Rr]equest" SYMBOL_IDENTIFIER "?" MEDIA_TYPE "?[[:blank:]]*";
+    const char* const RequestRegex = "^[[:blank:]]*[Rr]equest([[:blank:]]" SYMBOL_IDENTIFIER ")?" MEDIA_TYPE "?[[:blank:]]*";
```

Comment:
This is more about being lenient with user's mistakes. We do this at a lot of places. If you look at https://github.com/apiaryio/snowcrash/blob/master/src/AttributesParser.h#L20, you can see it allows `Attribute(s)`.